$MirOS: ports/comms/linphone/patches/patch-coreapi_linphonecore_c,v 1.1.7.1 2005/03/18 15:43:50 tg Exp $
--- coreapi/linphonecore.c.orig	Tue Oct  5 23:54:07 2004
+++ coreapi/linphonecore.c	Tue Oct  5 23:54:41 2004
@@ -35,9 +35,7 @@ Foundation, Inc., 59 Temple Place - Suit
 #include <netinet/in.h>
 #include <arpa/inet.h>
 
-#ifdef INET6  
 #include <netdb.h>  
-#endif
 
 #define MAX_IF 20
 const double bandwidths[5]=
@@ -1145,7 +1143,7 @@ gint linphone_core_set_sound_device(Linp
 	}
 	sndcard=snd_card_manager_get_card(snd_card_manager,devid);
 	if (sndcard==NULL){
-		g_warning("Sound card with index %i does not exist.");
+		g_warning("Sound card does not exist.");
 		devid=0;
 	}
 	lc->sound_conf.dev_id=devid;
